### ISSUE TITLE
final peer review dmt228

### ISSUE THREAD

**Issue by Ting-Kerbell** [meredithdobrzynski/google_play_project#13]
This paper tackles the issue of figuring out what features makes a mobile app more popular in the form of high reviews. The authors used a dataset of almost 11k rows with numerous features such as type and price while predicting on the variable of rating. In terms of the data preprocessing, they corrected skew, filled missing values, removed certain features and outliers. They showed a correlation map for exploratory data analysis. They used models such as decision trees and least squares regression as well as used different tactics to prevent under and overfitting. The authors then evaluated the different models with different loss functions and parameters. They the discuss whether or not the model’s success is a weapons of Math Destruction as well as the fairness of it. 
In terms of strengths, the authors clearly applied good exploratory data analysis, feature engineering, and used a multitude of models and losses. The exploratory data analysis and breakdown of data types painted a clear picture of the data itself for the reader. The feature engineering applied makes sense and is good practice that I wish I had thought about doing in my project. The wide range of different techniques used is great and informative. Great job with the concepts from class!
In terms of the weaknesses in the paper, there was a lack of informative metrics, not enough qualitative model conclusions and the graphs were not exactly clear. In terms of the metrics, the authors used MSE and MAE to measure their success, but this does not paint a picture of how good the model is at predicting the rating. Metrics such as accuracy or F1 would have helped more.   Much of the model results are based on improving MSE, but it is sort of unclear in a qualitative way what features are more important for the model, so from a bigger picture its rather unclear what the results mean. In addition, the results would be a lot more informative relative to the original question in the paper if they used different graphs other than scatterplots between true and predicted rating.
